action/example_install_test.go

```diff
@@ -0,0 +1,126 @@
+package action_test
+
+import (
+	"time"
+
+	"github.com/cnabio/cnab-go/action"
+	"github.com/cnabio/cnab-go/bundle"
+	"github.com/cnabio/cnab-go/claim"
+	"github.com/cnabio/cnab-go/credentials"
+	"github.com/cnabio/cnab-go/driver"
+	"github.com/cnabio/cnab-go/driver/lookup"
+	"github.com/cnabio/cnab-go/utils/crud"
+)
+
+// Install the bundle and only record the success/failure of the operation.
+func Example_install() {
```

Comment:
nit: I think we've used `Example_Install` as convention for tests before (looking at, for example, `TestClaim_Validate`).

Reply:
Oh yeah so I really wanted to name it that too, but then it doesn't show up in the godoc! 😅 It only shows up when lower-cased. I'm not sure why but that's why I named the whole file examples that way.

In the "doc" for godoc, they tell you to look at the sort package and that's what they do there too

https://blog.golang.org/examples
https://golang.org/src/sort/example_keys_test.go

If you know how to get it working with upper-case, let me know!

Reply:
Have we tried `ExampleInstall`?

Reply:
Yup! godoc doesn't like that either 🤷‍♀️